ibm_catalog.json

```diff
@@ -0,0 +1,501 @@
+{
+	"products": [
+		{
+			"label": "Apptio Cloudability",
+			"name": "apptio-cloudability-onboarding",
```

Comment:
I think you will that for service framework, they will ask you to start the naming with `deploy-arch-ibm-` (unless you already got this name approved in partner centre?)